config-cmd/signin.go

```diff
@@ -0,0 +1,127 @@
+package configCmd
```

Comment:
Would it make more sense to have the signin command in a different package? As a new contributor, I probably wouldn't expect it to be defined with the config commands.